release notes/v0.53.0.md

```diff
@@ -0,0 +1,295 @@
+k6 `v0.53.0` is here 🎉! This release includes:
+
+- Native ECMAScript modules support
```

Comment:
```suggestion
- Native ECMAScript module support
```

Reply:
I prefer to keep it in plural as that is how it is used in most places.